subsys/fb/Kconfig

```diff
@@ -27,7 +45,7 @@ config CHARACTER_FRAMEBUFFER_SHELL
 	  console.
 
 if CHARACTER_FRAMEBUFFER_SHELL
-	config CHARACTER_FRAMEBUFFER_SHELL_DRIVER_NAME
+	config CHARACTER_FRAMEBUFFER__DISPLAY_DEV_NAME
```

Comment:
nitpick remove the redundant `_` in front of `DISPLAY_DEV_NAME`

Reply:
ok